src/Database/DriverInterface.php

```diff
@@ -132,6 +168,14 @@ public function disableForeignKeySQL(): string;
      */
     public function enableForeignKeySQL(): string;
 
+    /**
+     * Returns whether the driver supports the feature.
+     *
+     * @param string $feature Driver feature name
+     * @return bool
+     */
+    public function supports(string $feature): bool;
```

Comment:
Adding a new method to the interface in a minor release would be a BC break. So for now we just add the `@method` annotation and add the method to the interface in 5.0.

Reply:
This was accidental. I added the annotation then copied this when writing the docblock.